scss/civicrm/common/_buttons.scss

```diff
@@ -371,3 +371,38 @@ a {
     }
   }
 }
+
+button {
+  background: $brand-primary;
```

Comment:
this whole thing is pretty worrying.

So all this time we had the styling of the default buttons wrong? Then what about all the other buttons that have been styled so far? What's the difference between them? Are we duplicating styles here?

@boriszaydlin i can't see any difference in the before/after screenshots re the buttons. And this is such a big change that it deserves a separate section in the PR with dedicated screenshots, please add it

Reply:
Seems we don't have any `<button>` as backstop didn't show any changes

Reply:
@boriszaydlin i don't know what you mean by "seems we don't have any button"

Can you give an answer to each of these questions
> So all this time we had the styling of the default buttons wrong? Then what about all the other buttons that have been styled so far? What's the difference between them? Are we duplicating styles here?

I don't understand how styling the `<button>` tag directly doesnt' produce any change. And if doesn't produce any change, what's the point of having this style at all?

Reply:
So all this time we had the styling of the default buttons wrong? - yes
Then what about all the other buttons that have been styled so far? - All other was not buttons but 'input type="button"' or 'input type="submit"'
What's the difference between them? - Their types I mentioned above
Are we duplicating styles here? - Color, background color, border-radius and text-transform
I don't understand how styling the <button> tag directly doesn't produce any change - It produce change to event manage page for Online Registration tab.
![buttons](https://user-images.githubusercontent.com/3272950/35628612-1d8f7778-06a5-11e8-8613-77b868e2960d.png)